### PR TITLE
ZN-350 - wdc vs-fw-activate-history command bug fix

### PR DESCRIPTION
**Pull request by AkhileshRN**

### CODE REVIEW

File: plugins/wdc/wdc-nvme.c
Comment (jeffreyalien) [inline plugins/wdc/wdc-nvme.c:4026]: There are certain FW versions that return Power On Hour in the hh:mm:ss format and others that return it in a timestamp format.  So I believe the cust_id check needs to remain here.

File: plugins/wdc/wdc-nvme.c
Comment (jeffreyalien) [inline plugins/wdc/wdc-nvme.c:4074]: Same comment as above - need to leave this code in to support FW that uses the Timestamp instead of Power on Hour.

**Review by jeffreyalien — CHANGES_REQUESTED** [review]
This pull request should not be merged as coded.  It will break the vs-fw-activate-history command for certain versions of WD drives.

**Comment by keithbusch** [comment]
Thanks Jeff, I'll wait for your review confirmation on this one before pulling.

File: plugins/wdc/wdc-nvme.c
Comment (AkhileshRN) [reply plugins/wdc/wdc-nvme.c:4026]: Addressed.

File: plugins/wdc/wdc-nvme.c
Comment (AkhileshRN) [reply plugins/wdc/wdc-nvme.c:4074]: Addressed.

**Review by jeffreyalien — APPROVED** [review]
The updated changes look good to me.  Keith feel free to merge this PR if you also agree.

**Pull request merged**
